### ISSUE TITLE
I want some details about tracking (not asking for code)

### ISSUE THREAD

**Issue by ArchNew** [Microsoft/human-pose-estimation.pytorch#71]
1. the size of the pose pool used in tracking.
2. the validation set is marked every other 4 frames. During testing, do you track all frames (including 4 unmarked frames between), or just those marked by the validation set?
